R/revision_analysis.R

```diff
@@ -0,0 +1,209 @@
+#' A function to describe revision behavior for an archive
+#' @description
+#' `revision_summary` removes all missing values, and then computes some basic
```

Comment:
"removes all misisng values, and then" --- reword or remove as this is `drop_nas`-dependent.